### PR TITLE
Add Monaco and xterm Theming

### PR DESCRIPTION
**Pull request by ShuchirJ**
This Pull Request adds support for monaco (the editor) and xterm (the console/shell) to also be colored.

Finalized model includes-
- An updated bookmarklet
- The actual package has been updated, so a new release would need to be pushed.
    - The YAML would contain a new section similar to `Styles` or `Meta`, called `Advanced`
    - monaco and xterm hex would be customized by adding the sections `monaco` and `xterm` similar to `styles` underneath `Advanced`
    - If these two values are present, they are pushed to the api to be uploaded to the DB
- The api has been updated to support these two values being added
- The `Theme` object for the api has been updated. 
    - The `.cereal()` method will add default monaco and xterm values as the default background variable if not customized separately.

### CODE REVIEW

**Comment by frissyn** [comment]
Looks good, need some time to migrate the Database to include the added fields. Pushing soon. Thanks!

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 frissyn
+Copyright (c) 2020 IreTheKID
```
Comment (BD103) [inline LICENSE]: Is this purposeful? Is the name "IreTheKID" or "Frissyn" preferred? @frissyn 

Also, shouldn't this be something along the lines of 2020-2021?

File: bin/make.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo -n "Theme Name >> "; read name
+
+if python "themes/$name/theme.py"; then
+    printf "\n\Theme '$name' made successfully.\n"
+else
+    printf "\n\nAn error occured building `$name` theme.\n"
+    exit 1
+fi
```
Comment (BD103) [inline bin/make.sh]: Add newline to end of file?

**Comment by spidunno** [comment]
> Looks good, need some time to migrate the Database to include the added fields. Pushing soon. Thanks!

just so you know, this code was basically directly taken from our codebase, there is a licensing field but just know what is going on.

**Comment by ShuchirJ** [comment]
@spidunno I've already credited your whole team with the MIT license whenever that code bit gets used.

For anyone wondering, the new Reflux engine uses spidunno's team's code bit for xterm theming.

**Comment by ShuchirJ** [comment]
Heya @frissyn I made specific changes so the values don't need to be in the db at all, just update the api server with the api folder when you do merge!

File: LICENSE (hunk 1 above)
Comment (ShuchirJ) [reply LICENSE]: Oh what the heck, my repl didn't pull properly, lemme fix it

**Comment by frissyn** [comment]
No commit emojis, yuck.

Looks good otherwise, thanks

**Pull request merged**